main/EDMainMod.F90

```diff
@@ -131,7 +132,7 @@ subroutine ed_ecosystem_dynamics(currentSite, bc_in)
     ! !LOCAL VARIABLES:
     type(ed_patch_type), pointer :: currentPatch
     integer :: el              ! Loop counter for elements
-
+    integer :: do_patch_dynamics ! for some modes, we turn off patch dynamics
```

Comment:
is this the best place to store this variable, or should it be in some place where flags in general get stored?

Reply:
I guess it's not a thing I'd want to pass from the HLM?  Perhaps in FatesGlobals, so if we needed to look at it somewhere else for some reason we could? What do think @rgknox ?

Reply:
I think its ok where it is, because it is local to just this routine right?